pennylane_cirq/simulator_device.py

```diff
@@ -132,9 +132,10 @@ def apply(self, operations, **kwargs):
         # pylint: disable=missing-function-docstring
         super().apply(operations, **kwargs)
 
-        # We apply an identity gate to all wires, otherwise Cirq would ignore
+        # We apply identity gates to all wires, otherwise Cirq would ignore
         # wires that are not acted upon
-        self.circuit.append(cirq.IdentityGate(len(self.qubits))(*self.qubits))
+        for q in self.qubits:
+            self.circuit.append(cirq.IdentityGate(1)(q))
```

Comment:
Not sure I understand why ``len(self.qubits)`` can be replaced by 1?

Reply:
In this case, it looks like you are applying `len(self.qubit)` single-qubit identity gates, instead of one large identity gate.

Reply:
Yes, but this is just putting a band-aid over an existing hack. Would be better for us to keep better track of the unused wires in the plugin (which we can now do easily!)

Reply:
Have added a TODO in the codebase